CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+cmake_minimum_required(VERSION 3.16)
+project(langocr)
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTORCC ON)
+set(CMAKE_AUTOUIC ON)
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_SOURCE_DIR})
+
+## Library source files
```

Comment:
There is a plan and desire to make sub-directory specific cmake build instructions, but that will come in on a different PR